client/components/banner/test/test.jsx

```diff
@@ -102,12 +107,12 @@ describe( 'Banner basic tests', () => {
 	} );
 
 	test( 'should render a <PlanPrice /> when price is specified', () => {
-		const comp = shallow( <Banner { ...props } price={ 100 } /> );
+		const comp = mount( <Banner { ...props } price={ 100 } /> );
 		assert.lengthOf( comp.find( 'PlanPrice' ), 1 );
 	} );
 
 	test( 'should render two <PlanPrice /> components when there are two prices', () => {
-		const comp = shallow( <Banner { ...props } price={ [ 100, 80 ] } /> );
+		const comp = mount( <Banner { ...props } price={ [ 100, 80 ] } /> );
 		assert.lengthOf( comp.find( 'PlanPrice' ), 2 );
```

Comment:
The following test `should render no <PlanPrice /> components when there are no prices`, should also use `mount` to be consistent I guess.

Reply:
Another way around this is to import `<PlanPrice />`  into the test, and then:

```
import PlanPrice from 'my-sites/plan-price';
....

	test( 'should render two <PlanPrice /> components when there are two prices', () => {
		const comp = mount( <Banner { ...props } price={ [ 100, 80 ] } /> );
		assert.lengthOf( comp.find( PlanPrice ), 2 );
	} );
```